liquid-staking/src/setup/delegation.rs

```diff
@@ -435,6 +426,10 @@ pub trait DelegationModule:
         contract_address: &ManagedAddress,
     ) -> SingleValueMapper<DelegationContractData<Self::Api>>;
 
+    #[view(getVoteContract)]
+    #[storage_mapper("voteContract")]
+    fn governance_contract(&self) -> SingleValueMapper<ManagedAddress>;
```

Comment:
Not really sure what is happening here. This is somewhat doubled with the storage in the GovernanceModule, but here the storage key is not good. Most probably is the same as with the vote_contract storage in the VoteModule, so delete it from here.